TELEMETRY.md

```diff
@@ -0,0 +1,43 @@
+# Telemetry in the CodeQL extension for VS Code
+
+If you specifically opt-in to permit GitHub to do so, GitHub will collect usage data and metrics for the purposes of helping the core developers to improve the CodeQL extension for VS Code. This data will not be shared with any parties outside of GitHub and will be retained for a maximum of 30 days.
```

Comment:
"for a maximum of 30 days" is not consistent with what is in the text below

> Aggregated data of command identifiers and run times will be retained for a maximum of 180 days.

Reply:
On further reflection, this is a bit of a problem. IP Address and the machine ID is not "anonymous". And we treat anonymous data differently. I'll have to clarify that.